deployments/admin-api/deploy.sh

```
@@ -0,0 +1,195 @@
+    if ! command -v docker &> /dev/null; then
+        echo "Error: docker is not installed"
+        exit 1
+    fi
+
+    echo "Prerequisites check passed"
+}
+
+# Function to build Docker image
+build_image() {
+    echo "Building Docker image..."
+    docker build -t ${APP_NAME}:${VERSION} -f deployments/admin-api/Dockerfile .
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)